recipe/build.sh

```diff
@@ -10,3 +9,16 @@ bash configure  --prefix=$PREFIX $OPTS
 make
 make check
 make install
+
+if [ "$(uname)" == "Darwin" ]; then
+    # Based on
+    # https://github.com/Homebrew/homebrew-core/blob/master/Formula/emacs.rb
+    mv nextstep/Emacs.app $PREFIX/Emacs.app
```

Comment:
Not that I'm aware of. Spyder seems to just use `python.app` and doesn't provide an App bundle of its own. I think most other Python things do something similar. So this is really the trend setter.

Sure, I don't really have a strong preference, but keeping Applications organized somewhere does seem sensible. So I guess `$PREFIX/Applications` sounds nice. Then again we don't really have anything else to keep organized ATM. Plus if putting it there causes issues with the running the App, then `$PREFIX` is certainly better.

Are they installing their own wrapper script to start emacs from the command line?